lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_new_milestone_action.rb

```diff
@@ -2,7 +2,6 @@
 require 'date'
 require_relative '../../helper/github_helper'
 require_relative '../../helper/ios/ios_version_helper'
-require_relative '../../helper/android/android_version_helper'
```

Comment:
This version helper is not being used in this action.

Reply:
A bit strange that this action is in `common/` yet use `ios_version_helper` (to call ` Fastlane::Helper::Ios::VersionHelper.calc_next_release_version(last_stone[:title])`)…
I mean, in practice I guess it doesn't matter because the computation for the `next_release_version` is the same for iOS and Android, which is why we ended up using an arbitrary one I guess, but still the unbalance feels a bit strange.

Not really a problem for this PR, and I'm guessing we'll plan to deprecate this action in a few months once everything have migrated to ReleaseV2 and that ReleaseV2 will be the one creating the new milestones instead of `Fastlane`… so not worth fixing anyway 🙃